python/solid_dmft/read_config.py

```diff
@@ -581,6 +589,11 @@
 
                                  'fixed_mu_value': {'converter': float, 'used': True, 'default': 'none'},
 
+                                 'calc_mu_method': {'valid for': lambda x, _: x in ['dichotomy', 'newton', 'brent'],
+                                                 'used': True,
+                                                 'default': 'dichotomy',
+                                                 },
+
```

Comment:
I'd personally be in favor of setting the default to brent but that would of course change the default behavior from before.